src/Apps/Sell/Routes/DetailsRoute.tsx

```diff
@@ -52,6 +61,12 @@ export const DetailsRoute: React.FC<DetailsRouteProps> = props => {
     medium: submission.medium ?? "",
   }
 
+  const yearInputRef = React.useRef<HTMLInputElement | null>(null)
+
+  useEffect(() => {
+    yearInputRef.current?.focus()
+  }, [])
```

Comment:
Focusing the year input when the user enters the screen.

Reply:
This seems like something we should make a broad UX decision about across steps. When navigating to a step, should we auto-focus on the first input? Would we want that behavior on both desktop and mobile web? I feel like this might result in a subpar experience on mWeb where the keyboard would automatically raise in response to an active input. This might limit our ability to take advantage to utilize the full screen to communicate any framing/guidance on the step.